client_code/session.py

```diff
@@ -14,6 +16,24 @@ def tabulator_options():
     }
 
 
+def sync_storage(scroll_store, local_store):
+    scroll_ballots = scroll_store.all()
+    local_ballots = local_store.all(Ballot)
+    for ballot in scroll_ballots:
+        if ballot.uuid not in [b.uuid for b in local_ballots]:
+            local_store.save(ballot)
+
+
+def sync_ballot(scroll_store, local_store, ballot_id):
```

Comment:
I think this should return the Ballot instance

Reply:
yeah i debated with this